internal/web_api.go

```diff
@@ -95,8 +104,14 @@ func (a *api) Call(action string, reqPayload interface{}, respPayload interface{
 		}
 		req.Header.Set(key, val[0])
 	}
+	err = a.send(req, respPayload)
 
-	return a.send(req, respPayload)
+	executionTime := time.Now().Sub(start)
+	if a.statsSink != nil {
+		a.statsSink(metrics.APICallStats{action, executionTime, err == nil})
+	}
```

Comment:
set `a.StatsSink` to no-op func in the constructor of API and don't check for `nil` here